kubernetes/yaml/base/jump-pad.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
 spec:
   containers:
   - name: jump-pad
-    image: dpcpinternal/jump-pad:main-d16defe-01.Sep.2021.23.47.21
+    image: gcr.io/velero-gcp/kibishii-jump-pad:main-d16defe-01.Sep.2021.23.47.21
```

Comment:
Image name should be modify in a new PR after make command execution in main branch manually.

Reply:
Got it. I will remove changes in this file.